sdk/servicebus/service-bus/test/perf/rhea-promise/receive.ts

```diff
@@ -70,11 +81,15 @@ async function RunTest(
     name: "receiver-1",
     source: {
       address: entityPath
-    }
+    },
+    credit_window: 0
   });
 
-  receiver.on(ReceiverEvents.message, async (context) => {
-    //console.log("Received message: %O", context.message);
+  receiver.addCredit(maxConcurrentCalls);
+
+  receiver.on(ReceiverEvents.message, async (context: any) => {
+    // console.log("Received message: %O", context.message.body);
+    receiver.addCredit(1);
```

Comment:
This should be in `else` block of the `if` condition below

Reply:
I moved it to the else block.
But it doesn't really matter since the code inside if block is executed only once (that too at the end of the test).